qiita_ware/processing_pipeline.py

```diff
@@ -22,51 +22,56 @@
 
 def _get_qiime_minimal_mapping(prep_template, out_dir):
     """Generates a minimal QIIME-compliant mapping file for split libraries
-
```

Comment:
Why did you remove this line? numpy docs allow them, same for the next two deleted lines.